i18n/es.json

```diff
@@ -232,12 +232,12 @@
     "translation": "No se puede remover el usuario del canal predeterminado {{.Channel}}"
   },
   {
-    "id": "api.channel.remove_member.removed",
-    "translation": "%v removido del canal."
+    "id": "api.channel.remove_channel_member.type.app_error",
+    "translation": "No se puede remover el usuario del canal."
```

Comment:
I have reviewed the translations to replace "remover" with "eliminar" in all places (in the translations server). "Remover" sounds like an anglicism, meanwhile "eliminar" sounds more natural (at least in spain).